apps/web/modules/users/views/users-public-view.getServerSideProps.tsx

```diff
@@ -3,7 +3,6 @@ import type { GetServerSideProps } from "next";
 import { encode } from "querystring";
 import type { z } from "zod";
 
-import { handleUserRedirection } from "@calcom/features/booking-redirect/handle-user";
```

Comment:
Changes on this file are to remove previous feature of profile forwarding, no longer needed.